### ISSUE TITLE
HT review - final plot

### ISSUE THREAD

**Issue by sebdalgarno** [bcgov/shinyrems#88]
When manually add guideline, guideline line starts at a time period preceding the data date period entered.
Guideline doesn't remove itself when delete number in guideline box
Guideline still showing up as a separate facet sometimes
·          Error message pop up from calculating WQG from data. Not sure what means:
`x` and `y` must share the same src, set `copy` = TRUE (may be slow).
 'Rename Stations' creates two new plots above the non-renamed plots.

**Comment by HeatherGranger** [comment]
@sebdalgarno 
After a fresh install of wqbc and poisson shinyrems and a restart I replicated the two issues you mentioned in you email. 

I was able to replicate: When manually add guideline, guideline line starts at a time period preceding the data date period entered.
This occurs when I've looked at Station: Shuswap Lk TB #5, Phosphorus Total in the all years dataset. When I get to the plot tab, adjust the date range (I just wanted to view data from 2009), the plot adjusts, but when I manually add the guideline, the guideline line goes back to 1987 (the beginning of the dataset obtained in the data tab) although the plot is still only showing data point from 2009 onwards. 

I was also able to replicate: · Error message pop up from calculating WQG from data. Not sure what means:
x and y must share the same src, set copy = TRUE (may be slow).
I used same station as above, all years dataset, Arsenic Total. In the plot tab, I added a calculated long-term daily guideline and got the pop up with this error message. I did not adjust the dates for this. 

If you have any more questions, @JessicaPenno can test!
